scripts/doc_checker.py

```diff
@@ -268,13 +269,13 @@ def get_noqa_checks(doc: Docstring) -> list:
     else:
         # pydocstyle only check first line; aling with it
         noqa_str = source.split("\n", 1)[0]
-    if "noqa" not in noqa_str:
-        return []
 
-    if "noqa:" in noqa_str:
+    if "# noqa:" in noqa_str:
         noqa_checks = noqa_str.split("noqa:")[1].split(",")
```

Comment:
```suggestion
        noqa_checks = noqa_str.split("# noqa:", 1)[1].split(",")
```
otherwise stuff like
```python
def foo(param="noqa: FOO"): # noqa: hey
    return 10
```
would break